source/2_platform/rex_opengl/src/opengl_renderer.cpp

```diff
@@ -0,0 +1,145 @@
+#include "rex_renderer_core/renderer.h"
+
+#include "rex_opengl/opengl_error.h"
+
+#include "rex_std/string.h"
+
+#if REX_PLATFORM_X64
+#include <glad/gl.h>
+#else
+#error "Unsupported platform"
+#endif
+
+namespace rex
+{
+  // these are required for platform specific gl implementation calls.
+  extern bool rex_make_gl_context_current();
+  extern void rex_gl_swap_buffers();
+
+  //-------------------------------------------------------------------------
+  enum ClearBits
+  {
+    REX_CLEAR_COLOUR_BUFFER = GL_COLOR_BUFFER_BIT,
+    REX_CLEAR_DEPTH_BUFFER = GL_DEPTH_BUFFER_BIT,
+    REX_CLEAR_STENCIL_BUFFER = GL_STENCIL_BUFFER_BIT,
+  };
+
+  //-------------------------------------------------------------------------
+  struct ClearState
+  {
+    f32 rgba[4];
+    f32 depth;
+    u8  stencil;
+    u32 flags;
+  };
+
+  RendererInfo s_renderer_info;
+  ClearState s_clear_state{ 0.0f, 0.3f, 0.2f, 1.0f, 1.0f, 0x00, REX_CLEAR_COLOUR_BUFFER };
+
+  namespace renderer
+  {
+    //-------------------------------------------------------------------------
+    // general accessors
+    const RendererInfo& get_info()
```

Comment:
we don't use get_ for getters

Reply:
This is changed in a different commit IIRC